smoke-testNuGet.Config

```diff
@@ -3,6 +3,14 @@
   <packageSources>
     <!--To inherit the global NuGet package sources remove the <clear/> line below -->
     <clear />
+    <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
```

Comment:
I'm not sure why we want these sources in the smoke tests?

Reply:
I think the auto-update must grab `*NuGet.config*` or something, this wasn't from me.  I'll remove it and verify smoke tests still work.